emva1288/process/results.py

```diff
@@ -636,16 +636,15 @@ def u_I_var_DN(self):
         if fit[0] < 0:
             return np.nan
         # Multiply by 10^9 because exposure times are in nanoseconds
-        return fit[0] * (10 ** 9)
+        return fit[0] / self.K * 1e9
```

Comment:
U_I is in [e/s] and K is in [DN/e] why divide by K and not multiply?

Reply:
Formula (20) from the EMVA1288 standard, release 3.1 states:
σ²<sub>d</sub> = σ²<sub>d,0</sub> + μ<sub>I</sub>t<sub>exp</sub>

The terms are expressed in electrons, but the code performs the fit in units of DN. So we rewrite the formula above using  σ<sub>y,dark</sub> = Kσ<sub>d</sub> (with K in [DN/e]):
σ²<sub>y,dark</sub> = K²σ²<sub>d</sub> = K²σ²<sub>d,0</sub> + K²μ<sub>I</sub>t<sub>exp</sub>

The slope of our fit (fit[0]) is the coefficient before t<sub>exp</sub>, so:
fit[0] = K²μ<sub>I</sub> or μ<sub>I</sub> = fit[0]/K²

We need to return μ<sub>I,DN</sub> = Kμ<sub>I</sub>, so:
μ<sub>I,DN</sub> = fit[0]/K.

This can be easily tested with real or simulated image data (e.g. from the `camera` module); only the division by K will yield the right value for the dark current in DN/s as estimated from the variance. You can check it by either comparing to the dark current value extracted from the mean (for real image data) or to the value passed to the `camera` (for simulated image data).

Reply:
@thomasdob thanks, I think you're right again, I was looking for the mismatch for a long time.
@mwuetschner what do you think?

Reply:
@JF1008 I think @thomasdob is right what do you think?

Reply:
@JF1008 I am ready to merge, are you OK with this?

Reply:
I also think he is right